website/docs/d/cbr_rule.html.markdown

```diff
@@ -0,0 +1,82 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_cbr_rule"
+description: |-
+  Get information about cbr_rule
+subcategory: "Context Based Restrictions"
+---
+
+# ibm_cbr_rule
+
+Provides a read-only data source for cbr_rule. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
+data "ibm_cbr_rule" "cbr_rule" {
+	rule_id = "rule_id"
+}
+```
+
+## Argument Reference
+
+Review the argument reference that you can specify for your data source.
+
+* `rule_id` - (Required, Forces new resource, String) The ID of a rule.
+  * Constraints: The maximum length is `32` characters. The minimum length is `32` characters. The value must match regular expression `/^[a-fA-F0-9]{32}$/`.
+
+## Attribute Reference
+
+In addition to all argument references listed, you can access the following attribute references after your data source is created.
+
+* `id` - The unique identifier of the cbr_rule.
+* `contexts` - (Required, List) The contexts this rule applies to.
+  * Constraints: The maximum length is `1000` items. The minimum length is `1` item.
+Nested scheme for **contexts**:
+	* `attributes` - (Required, List) The attributes.
+	  * Constraints: The minimum length is `1` item.
+	Nested scheme for **attributes**:
+		* `name` - (Required, String) The attribute name.
```

Comment:
these are all computed attributes..
Just mention the type of attributed.. Remove required from all attributes

Reply:
done